scribus/plugins/tools/hunspellcheck/hunspellpluginimpl.cpp

```diff
@@ -181,7 +181,7 @@ bool HunspellPluginImpl::parseTextFrame(StoryText *iText)
 			wf.replacements = hspellerMap[wordLang]->suggest(word);
 			wordsToCorrect.append(wf);
 		}
-		currPos = iText->nextWord(wordStart);
+		currPos = iText->endOfWord(wordStart);
```

Comment:
In that case I will make nextWord call endOfWord because they will be same just duplicated code. Is that right?